qcengine/programs/nwchem/harvester.py

```diff
@@ -217,18 +217,19 @@ def harvest_outfile_pass(outtext):
                 psivar[f'{cc_corr} CORRECTION ENERGY'] = mobj.group(1)
                 psivar[f'{cc_plain} CORRELATION ENERGY'] = mobj.group(2)
                 psivar[f'{cc_plain} TOTAL ENERGY'] = mobj.group(3)
-        #Other TCE
-        #mobj = re.findall(
-        #        r'^\s+' + r'Iterations converged' + r'\s*' +
-         #       r'^\s+' + r'(.*?)' + r' correlation energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*'
-         #       r'^\s+' + r'(.*?)' + r' total energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*$', 
-          #      outtext, re.MULTILINE | re. DOTALL)
+        #Process other TCE cases
+        for cc_name in [r'CISD', r'CISDT', r'CISDTQ', r'CCD', r'CCSD', r'CCSDT', r'CCSDTQ', r'LCCSD', r'LCCD']:
+            mobj = re.findall(
+                r'^\s+' + r'Iterations converged' + r'\s*' +
+                r'^\s+' + cc_name + r' correlation energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*'+
+                r'^\s+' + cc_name + r' total energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*$', 
+                outtext, re.MULTILINE)
 
-        #for mobj_list in mobj:
-         #   print('matched %s' % mobj_list[0])
-         #   print(mobj_list)
-         #   psivar['%s CORRELATION ENERGY' % mobj_list[0]] = mobj_list[1]
-         #   psivar['%s TOTAL ENERGY' % mobj_list[2]] = mobj_list[3]
+            for mobj_list in mobj:
+                print('matched %s'% cc_name)
+                print(mobj)
+                psivar[f'%s CORRELATION ENERGY' % cc_name] = mobj_list[0]
```

Comment:
can use f-strings like the blocks above. and just `if mobj` when back to `re.search`